packages/cli/src/commands/proxy.ts

```diff
@@ -0,0 +1,46 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { pick } from 'lodash';
+import { CommandModule } from 'yargs';
+import { createMultiProcessPrism, CreateProxyServerOptions, createSingleProcessPrism } from '../util/createServer';
+import sharedOptions from './sharedOptions';
+
+const proxyCommand: CommandModule = {
+  describe: 'Start a proxy server with the given spec file',
+  command: 'proxy <spec> <upstream>',
+  builder: yargs =>
+    yargs
+      .positional('spec', {
```

Comment:
Can we talk about "document" as spec could mean "the local file" or could mean "OpenAPI v2, v3, JSON Schema" etc.

Reply:
I'm sorry I keep doing this mistake.